src/abi_impl.rs

```diff
@@ -79,7 +79,11 @@ fn call_module(env: &Env, address: &Address, function: &str, param: &str) -> ABI
 }
 
 /// Transfer an amount from the address on the current call stack to a target address.
-pub(crate) fn transfer_coins(env: &Env, to_address: i32, raw_amount: i64) -> ABIResult<()> {
```

Comment:
Why did you rename it ?

Reply:
we can correct this in a followup if you think it is a mistake